components/Product/models.rst

```diff
@@ -1,71 +1,199 @@
+
+.. _component_product_model_archetype:
+
+Archetype
+---------
+
+.. note::
+   This model extends the :ref:`component_archetype_model_archetype`
+   and implements the :ref:`component_product_model_archetype-interface`. |br|
+   For more detailed information go to `Sylius API Archetype`_.
+
+.. _Sylius API Archetype: http://api.sylius.org/Sylius/Component/Product/Model/Archetype.html
+
+.. _component_product_model_archetype-translation:
+
+ArchetypeTranslation
+--------------------
+
+.. note::
+   This model extends the :ref:`component_archetype_model_archetype-translation`
+   and implements the :ref:`component_product_model_archetype-translation-interface`. |br|
+   For more detailed information go to `Sylius API ArchetypeTranslation`_.
+
+.. _Sylius API ArchetypeTranslation: http://api.sylius.org/Sylius/Component/Product/Model/ArchetypeTranslation.html
+
+.. _component_product_model_attribute:
+
+Attribute
+---------
+
+.. note::
+   This model extends the :ref:`component_attribute_model_attribute`
+   and implements the :ref:`component_product_model_attribute-interface`. |br|
+   For more detailed information go to `Sylius API Attribute`_.
+
+.. _Sylius API Attribute: http://api.sylius.org/Sylius/Component/Product/Model/Attribute.html
+
+.. _component_product_model_attribute-translation:
+
+AttributeTranslation
+--------------------
+
+.. note::
+   This model extends the :ref:`component_attribute_model_attribute-translation`
+   and implements the :ref:`component_product_model_attribute-translation-interface`. |br|
+   For more detailed information go to `Sylius API AttributeTranslation`_.
+
+.. _Sylius API AttributeTranslation: http://api.sylius.org/Sylius/Component/Product/Model/AttributeTranslation.html
+
+.. _component_product_model_option:
+
+Option
+------
+
+.. note::
+   This model extends the :ref:`component_variation_model_option`
+   and implements the :ref:`component_product_model_option-interface`. |br|
+   For more detailed information go to `Sylius API Option`_.
+
+.. _Sylius API Option: http://api.sylius.org/Sylius/Component/Product/Model/Option.html
+
+.. _component_product_model_option-value:
+
+OptionValue
+-----------
+
+.. note::
+   This model extends the :ref:`component_variation_model_option-value`
+   and implements the :ref:`component_product_model_option-value-interface`. |br|
+   For more detailed information go to `Sylius API OptionValue`_.
+
+.. _Sylius API OptionValue: http://api.sylius.org/Sylius/Component/Product/Model/OptionValue.html
+
+.. _component_product_model_option-translation:
+
+OptionTranslation
+-----------------
+
+.. note::
+   This model extends the :ref:`component_variation_model_option-translation`
+   and implements the :ref:`component_product_model_option-translation-interface`. |br|
+   For more detailed information go to `Sylius API OptionTranslation`_.
+
+.. _Sylius API OptionTranslation: http://api.sylius.org/Sylius/Component/Product/Model/OptionTranslation.html
```

Comment:
I know it sucks to say that - I think we should remove all Extended Models doc here. Not giving us much more (or any?) info than api.sylius.org. :)